.github/workflows/ruff.yml

```
@@ -0,0 +1,14 @@
+name: Ruff
+
+on:
+  push:
+    branches: [add-github-workflows]
+  pull_request:
+    branches: [add-github-workflows]
```

Not sure if we won't need the `*` here to trigger it (push for main should be fine, but pull requests can come from any branch)

```suggestion
    branches: 
      - "*"
```

---

Same for unit tests

---

Yeah, this was part of testing (but it didn't work :(). Originally, I had there `main` branch